sdk/eventhub/azure-eventhubs/azure/eventhub/aio/producer_async.py

```diff
@@ -110,6 +111,10 @@ def __init__(  # pylint: disable=super-init-not-called
             await self._connect()
             self.running = True
 
+            self._max_message_size_on_link = self._handler.message_handler._link.peer_max_message_size if\
+                self._handler.message_handler._link.peer_max_message_size\
+                else constants.MAX_MESSAGE_LENGTH_BYTES
```

Comment:
Minor: This statement is a bit of mouthful - I think it would be cleaner just to do it with an "or" statement:
```
self._max_message_size = self._handler.message_handler._link.peer_max_message_size or constants.MAX_MESSAGE_LENGTH
```

Also I think it will need a pylint disable for protected-access.

Reply:
good suggestion. Fixed

Reply:
You fixed it in the sync class but not the async one :)